Source/RapyutaSimulationPlugins/Private/Core/RRGameSingleton.cpp

```diff
@@ -62,7 +83,7 @@ URRGameSingleton* URRGameSingleton::Get()
     return singleton;
 }
 
-bool URRGameSingleton::InitializeResources()
+bool URRGameSingleton::InitializeResources(bool bInRequestResourceLoading)
```

Comment:
When you use this with bInRequestResourceLoading=false ?

Reply:
As noted in the description (pls check it first), it is an option for users to avoid early loading `uasset` in DynamicContents if there is no need for it (eg in a pure BP-based application)

Reply:
My point is that there is no use case for it yet but for future use case?

Reply:
ah yes,  It is kept here for the mentioned use case, with the purpose to make RRGameSingleton less coupling. In cmdlet I used to experiment with it but still ended up requiring runtime resources.